README.md

```diff
@@ -1 +1,168 @@
-# redirects
\ No newline at end of file
+# Url Redirects with GitHub
+
+_Counterpart to the [redirects-website repo](../../../redirects-website)._
+
+✏️ [How to edit](#how-to-edit)  
+💡 [Motivation](#motivation)  
+✔️ [Comparison](#comparison)  
+🧠 [How it works](#how-it-works)  
+⚙️ [Setup](#setup)
+
+## How to edit
+
+1. Add/change/remove redirect entries in one or more [`.yaml` files in the top folder](../../blob/main/redirects.yaml).
+2. Commit the changes to the `main` branch.
+3. Changes should take effect automatically within a minute or so.
+   Verify that no errors occurred in the automatic process here: [![Encode and deploy](../../actions/workflows/deploy.yaml/badge.svg)](../../actions/workflows/deploy.yaml)
+
+You can do this [directly on github.com](../../edit/main/redirects.yaml) (tip: press <kbd>.</kbd> right now), or locally with git.
+
+## Motivation
+
+You've likely heard of a service called [Bitly](https://bitly.com/).
+It allows you to convert a long link like `some-website.com/a-long-url?search=a-bunch-of-characters` into a shorter one like `bit.ly/98K8eH`.
+When someone visits the shorter link, Bitly automatically "redirects" them to the longer one.
+You can think of it like a shortcut.
+
+Bitly also offers several other features that you probably want:
+
+- You can customize the text after the `/`, giving you a url that a human could actually remember and type in manually, like `bit.ly/MyCoolLink` (they call these "back-halves").
+- You can set up a custom domain to brand your links the way you want, giving you an even nicer and easier-to-use link, like `my-website.com/MyCoolLink`.
+- You can see how many and what kinds of people have used the link over time (i.e. analytics).
+
+This all sounds great, so what's the problem?
+Well namely: _Bitly [hides or limits a lot of this functionality behind a paywall](https://bitly.com/pages/pricing)_, and their free plan has become more and more limited over time.
```

Comment:
After reading this section I felt that Bitly might be solely blamed for a larger issue with all url-shortener businesses. Providing the barebones suggestion below to help broaden this statement. Tying in other examples within the readme may also make the content more understandable or searchable for someone who may not use Bitly. 

```suggestion
Well namely: URL shorteners like Bitly  _[hide or limit a lot of functionality behind paywalls](https://bitly.com/pages/pricing)_, and their free plans are sometimes changed to be more limiting.
```

Reply:
In my research, I found Bitly free plan to be the most limited by a significant margin (maybe others would have different findings). That, and the fact that they are the biggest name, made me single them out.

I also have a sentence about other competing services below that which I've tweaked the wording of to hopefully make it more fair/clear.

Reply:
Actually I'm rewriting this section to list Bitly and some of its alternatives right at the start, and mention things more generally afterward.